app/server/api/user_api.py

```diff
@@ -210,10 +210,13 @@ def post(self, user_id):
 
         if reset_user_id is not None:
             user = User.query.get(reset_user_id)
-
             if user is None:
                 return make_response(jsonify({'message': 'No user found for ID: {}'.format(reset_user_id)})), 404
 
+            user.set_one_time_code(None)
```

Comment:
i thought resetting pin was re-applying a new on-time-code?

Reply:
It does-- when supplied_one_time_code is `None`, it generates a new one